api/emAnno/emAnnoService.ts

```diff
@@ -45,7 +46,6 @@ export async function handlePost(annotationsPath: string, body: Annotation, req:
         logger.error(e.message)
```

Comment:
Again not sure if it was implemented by @adnanakgun but we need to protect against the assumption that 'message' exists on the error?

Reply:
There was a hotfix on master that was eventually reverted:
`e.message ? logger.error(e.message) : logger.error('Error in em-anno-service get response')` note that this for a `get` method

Same change was also applied to redaction service

Reply:
okay, well as this is going into develop it should be fine for now, but that also means we have ```e.message``` code in master? This was the source of P1 issue last time, so as long as we fix in both branches before a new deployment it should be okay for now